central/detection/service/service_impl.go

```diff
@@ -136,9 +136,15 @@ func (s *serviceImpl) DetectBuildTime(ctx context.Context, req *apiV1.BuildDetec
 	img := types.ToImage(image)
 
 	enrichmentContext := enricher.EnrichmentContext{}
+	if req.GetNoExternalMetadata() && req.GetForce() {
```

Comment:
Nit: GetNoExternalMetadata is a purely code construct, we should describe what that means. e.g. incompatible with not fetching metadata from external sources